src/main/java/com/github/javafaker/Lorem.java

```diff
@@ -32,15 +32,15 @@ public String characters(boolean includeUppercase) {
     }
 
     public String characters(int minimumLength, int maximumLength) {
-        return characters(faker.random().nextInt(maximumLength - minimumLength) + minimumLength, false);
+        return characters(faker.random().nextInt(maximumLength - minimumLength + 1) + minimumLength, false);
```

Comment:
worth having a test that prove these work now?

Reply:
Yeah...this just need to test when `maximumLength = minimumLength` the length of password equals to 'maximunLength'. I attach my test file below.